base/applications/notepad/main.c

```diff
@@ -649,6 +651,14 @@ int WINAPI _tWinMain(HINSTANCE hInstance, HINSTANCE prev, LPTSTR cmdline, int sh
 
     while (GetMessage(&msg, 0, 0, 0))
     {
+        /* Make F3 key effective on Find/Replace dialog */
```

Comment:
Why is this hack needed? Shouldn't this be handled correctly by the IsDialogMessage+TranslateAccelerator + DispatchMessage?

Reply:
The target window is different from the main window, so the `CMD_SEARCH_NEXT` command won't reach the main window.

Reply:
In standard notepad, (and notepad++ etc.), pressing F3 while the find dialog is focused, does not work (one needs to press e.g. ENTER on the "Search next" button).

Reply:
Fixed the calling order of `TranslateAccelerator` and `IsDialogMessage`.